bin/crew

```diff
@@ -1127,24 +1127,35 @@ def install_package(pkgdir)
       end
     end
     if File.exist?("#{CREW_PREFIX}/bin/rdfind")
-        puts "Using rdfind to convert duplicate files to hard links."
-        system "#{CREW_PREFIX}/bin/rdfind -removeidentinode true -makesymlinks false -makehardlinks true -makeresultsfile false ."
+      puts 'Using rdfind to convert duplicate files to hard links.'
+      system 'rdfind -removeidentinode true -makesymlinks false -makehardlinks true -makeresultsfile false .'
     end
-    if Dir.exists? "#{pkgdir}/home" then
-      if system("#{CREW_PREFIX}/bin/rsync -V > /dev/null 2>&1")
+
+    # check if the available rsync command support ACLs
+    rsync_available = File.exist?("#{CREW_PREFIX}/bin/rsync")
+    puts 'rsync is not working. Please install it with \'crew install rsync\''.lightred unless rsync_available
+
+    if Dir.exists? "#{pkgdir}/#{HOME}" then
+      if rsync_available
         system "rsync -ahHAXW --remove-source-files ./#{HOME.delete_prefix('/')}/ #{HOME}"
       else
-        puts 'rsync is not working. Please install it with \'crew install rsync\''.lightred if @opt_verbose
-        system "tar -c#{@verbose}f - ./usr/* ./home/* | (cd /; tar xp --keep-directory-symlink -f -)"
+        system "cp -a#{@verbose} ./#{HOME}/* #{HOME}"
```

Comment:
I'm VERY hesitant to change line 1138.  What happened to `./usr/*`?

Reply:
Oh, I see now.  The `./usr/*` files are handled below.

Reply:
The reason of `tar` was replaced by `cp` is `cp` is more suitable for copying files, the `-a` param is used to tell `cp` to perform like `tar` (keep owner/exec/group bit and don't touch symlinks)

Reply:
Apparently, tar is much faster than cp. See https://superuser.com/a/985080/1646144

But maybe that's old information?

Could you do some testing by seeing how long installing something large like gcc11 or llvm takes with the existing `tar` setup vs the `cp -a` setup you are suggesting?  I have no problem switching especially if it is faster.

Reply:
I imagine `mv` would be the fastest and most space efficient. :)

Reply:
Not necessarily! rsync can be faster...

Reply:
I'd be much more comfortable with the change from `tar` if an install with this were tested first.

Until that happens, I'd be ok with merging this with just the `rsync_available` changes and the fallback being the use of tar in both `crew` and `install.sh`.

Reply:
You are right,  `tar` is much more faster than `cp -a`, switching back to `tar`